src/recommendations/troubleshooting.md

```diff
@@ -1,15 +1,15 @@
 ---
 group: product-recommendations
-title: Troubleshooting Recommendations
+title: Troubleshoot Recommendations
 ---
 
-If after installing and configuring the module you are not seeing any recommendations, refer to the following list for possible solutions:
+If you have configured the `product-recommendations` module correctly, but you are not seeing any recommendations, try the following:
 
 -  It is possible that the module has not had enough time to collect behavioral data. Allow the system to run for 24 hours so it can start collecting data.
```

Comment:
One thing we can recommend here is to deploy a MLT rec type first (requires no behavioral data)